setup.cfg

```diff
@@ -83,6 +83,7 @@ extend-ignore = E203
 [mypy]
 ignore_missing_imports = True
 strict = True
+files = src
```

Comment:
Oops, didn't mean to commit this. Though it is handy for quick running (I usually add mypy to pre-commit)